source/1-SDLC-organization/lib/auto-bootstrap.sh

```diff
@@ -27,7 +27,8 @@ for ACCOUNT in $ACCOUNTS; do
     echo "Bootrapping account $ACCOUNT_NAME..."
     case $ACCOUNT_NAME in
         CICD)
-            npm run cdk bootstrap -- --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess --profile ${ACCOUNT_NAME}
+            # A custom template will apply PermissionsBoundary to CDK Execution role.
+            npm run cdk bootstrap -- --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess --template ./lib/cdk-bootstrap-template.yml --profile ${ACCOUNT_NAME}
```

Comment:
"--cloudformation-execution-policies" cannot be empty.  I keep the original AdministratorAccess while put permission boundary via "--template" to limit its privilege.

Reply:
cannot be empty due to CLI restriction ? or due to template parameters requirements ? If it's the later, since we control the template, we should be able to avoid adding Admin managed policy and just build our own. can't we ?